Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -152,10 +152,10 @@ def find_launchctl_with_wildcard(search)
         regex = Regexp.escape(search).gsub("\\*", ".*")
         system_command!("/bin/launchctl", args: ["list"])
           .stdout.lines.drop(1) # skip stdout column headers
-          .map do |line|
+          .map { |line|
```

Comment:
Sorry, really don't agree with this change. Would rather we enforced specs to use this format than change non-spec code for this.

Reply:
So you want `EnforcedStyle: line_count_based` (the default) everywhere (so for multi-line blocks they're always `do ... end`?

Reply:
@issyl0 Yes I think so.

Reply:
Done!